<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HumanMeasureSystem</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0" rowspan="3">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QwtDial" name="Dial_left_hip">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="lowerBound">
         <double>-180.000000000000000</double>
        </property>
        <property name="upperBound">
         <double>180.000000000000000</double>
        </property>
        <property name="scaleMaxMajor">
         <number>20</number>
        </property>
        <property name="scaleMaxMinor">
         <number>2</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="minScaleArc">
         <double>0.000000000000000</double>
        </property>
        <property name="maxScaleArc">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QwtDial" name="Dial_right_knee">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="lowerBound">
         <double>-180.000000000000000</double>
        </property>
        <property name="upperBound">
         <double>180.000000000000000</double>
        </property>
        <property name="scaleMaxMajor">
         <number>20</number>
        </property>
        <property name="scaleMaxMinor">
         <number>2</number>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
        <property name="totalSteps">
         <UInt>100</UInt>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="minScaleArc">
         <double>0.000000000000000</double>
        </property>
        <property name="maxScaleArc">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QwtDial" name="Dial_right_hip">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="lowerBound">
         <double>-180.000000000000000</double>
        </property>
        <property name="upperBound">
         <double>180.000000000000000</double>
        </property>
        <property name="scaleMaxMajor">
         <number>20</number>
        </property>
        <property name="scaleMaxMinor">
         <number>2</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="minScaleArc">
         <double>0.000000000000000</double>
        </property>
        <property name="maxScaleArc">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QwtDial" name="Dial_left_knee">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="lowerBound">
         <double>-180.000000000000000</double>
        </property>
        <property name="upperBound">
         <double>180.000000000000000</double>
        </property>
        <property name="scaleMaxMajor">
         <number>20</number>
        </property>
        <property name="scaleMaxMinor">
         <number>2</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="minScaleArc">
         <double>0.000000000000000</double>
        </property>
        <property name="maxScaleArc">
         <double>360.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLCDNumber" name="lcdNumber_left_hip"/>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="lcdNumber_right_hip"/>
      </item>
      <item row="3" column="0">
       <widget class="QLCDNumber" name="lcdNumber_left_knee"/>
      </item>
      <item row="3" column="1">
       <widget class="QLCDNumber" name="lcdNumber_right_knee"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QLCDNumber" name="lcdNumber_back">
          <property name="digitCount">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="lineEdit_lpmsPortName">
          <property name="text">
           <string>00:06:66:4B:24:D2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="comboBoxIMU_left_thigh">
        <item>
         <property name="text">
          <string>COM10</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxIMU_right_thigh">
        <item>
         <property name="text">
          <string>COM11</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QComboBox" name="comboBoxIMU_left_shank">
        <item>
         <property name="text">
          <string>COM12</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBoxIMU_right_shank">
        <item>
         <property name="text">
          <string>COM13</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_deviceConnect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLCDNumber" name="lcdNumber_right_shank">
        <property name="digitCount">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLCDNumber" name="lcdNumber_left_shank">
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="digitCount">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="lcdNumber_right_thigh">
        <property name="digitCount">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLCDNumber" name="lcdNumber_left_thigh">
        <property name="digitCount">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="2" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_username">
        <property name="text">
         <string>William</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_newFile">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Aharoni</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>NEW</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_saveFile">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Aharoni</family>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>SAVE</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
